mysql-test/suite/sys_vars/t/replicate_do_db_basic.test

```diff
@@ -1,4 +1,8 @@
---source include/not_embedded.inc
+--source include/have_sequence.inc
+# have show_slave_status
+--source include/have_log_bin.inc
```

Comment:
We shouldn't need `have_log_bin.inc` for `SHOW SLAVE STATUS`. A slave can actually run without binlogging :)